gcs/gcs.go

```diff
@@ -98,7 +102,7 @@ func BuildGCSFilter(P uint8, key [KeySize]byte, data [][]byte) (*Filter, error)
 	if len(data) == 0 {
 		return nil, ErrNoData
 	}
-	if len(data) > math.MaxInt32 {
+	if len(data) > math.MaxUint32 {
```

Comment:
IIRC we changed this as we had issues on 32-bit arm. I'll check to see if this compiles properly on my pi.

Reply:
You might be right -- the type of `math.MaxUint32` is an int, which is kind of silly. I'll rewrite this line to compare `uint64`s.